mlxtend/utils/counter.py

```diff
@@ -21,20 +21,49 @@ class Counter(object):
     start_newline : bool (default: True)
         Prepends a new line to the counter, which prevents overwriting counters
         if multiple counters are printed in succession.
+    precision: int (default: 0)
+        Sets the precison of the displayed iteration time.
```

Comment:
There's a little typo and it should be "precision" not "precison". However, I suggest changing the short description altogether ... E.g., the number 1234567.89 has a precision of 9, so it would be more "precise" (pun intended :P) to refer to it as:

    Sets the number of decimal places when displaying the time elapsed in seconds.

Reply:
Haha, very good, change made